lib/ws-kernel-picker.js

```diff
@@ -9,45 +9,49 @@ import Config from "./config";
 import * as services from "./jupyter-js-services-shim";
 import WSKernel from "./ws-kernel";
 
-class CustomListView extends SelectListView {
-  initialize(emptyMessage, onConfirmed) {
+class CustomListView {
+  constructor(emptyMessage, onConfirmed) {
     this.emptyMessage = emptyMessage;
     this.onConfirmed = onConfirmed;
-    super.initialize(...arguments);
-    this.storeFocusedElement();
+    this.previouslyFocusedElement = document.activeElement;
+    this.selectListView = new SelectListView({
+      itemsClassList: ["mark-active"],
```

Comment:
You might be right. I'll have to test without it later. My assumption was it had something to do with highlighting the selected item